### PR TITLE
Support for Red Hat 8 family

### PR DESCRIPTION
**Pull request by svendewindt**
An attempt to make this role compatible with Red Hat 8 family #20 . 
Not really sure this is the best way to solve this, the `/vars/RedHat8.yml` file looks like `/vars/Fedora.yml`.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - include_vars: "{{ item }}"
   with_first_found:
     - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}{{ ansible_distribution_version[:1] }}.yml"
```
Comment (ericzolf) [inline tasks/main.yml:7]: I guess that `ansible_distribution_major_version` would be a better choice?

File: vars/RedHat8.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# roles/rh-base/vars/RedHat.yml
```
Comment (ericzolf) [inline vars/RedHat8.yml:1]: It should read `RedHat8.yml`.

File: meta/main.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ galaxy_info:
     - name: EL
       versions:
         - 7
+        - 8
     - name: Fedora
       versions:
         - 29
```
Comment (ericzolf) [inline meta/main.yml:16]: I guess that you support more than just Fedora 29?!?

**Review by ericzolf — COMMENTED** [review]
I am hit by the missing RHEL8/CentOS8 support and would appreciate a quick update, hence I'm doing this review. Regarding your question regarding the best approach: Fedora might evolve differently (and faster) than RHEL/CentOS so I would keep them separate, even if they are more or less the same _today_.

**Comment by bertvv** [comment]
Implemented in 48b29e4

**Pull request closed without merging**